docs/2.8.0/docs/canton/usermanual/upgrading.rst

```diff
@@ -194,6 +194,64 @@ The ping command creates two contracts between the admin parties, then exercises
 Version Specific Notes
 ~~~~~~~~~~~~~~~~~~~~~~
 
+Upgrade to Release 2.8
+^^^^^^^^^^^^^^^^^^^^^^
+
+Version 2.8 extends the database schema. If you use the "migrate and start" feature, the database schema will be automatically updated.
+Otherwise, perform the manual database migration steps outlined in the :ref:`database migration steps <migrating_the_database>`.
+
+Explicit contract disclosure (Beta)
+"""""""""""""""""""""""""""""""""""
+The explicit contract disclosure feature is now in Beta and is enabled by default.
+Refer to the explicit contract disclosure :ref:`documentation <explicit-contract-disclosure>` for details.
+Note that the ``participant.ledger-api.explicit-disclosure-unsafe`` feature flag is deprecated and replaced by
+``participant.ledger-api.enable-explicit-disclosure``, set to true by default.
+
+Breaking configuration changes
+""""""""""""""""""""""""""""""
+
+**KMS wrapper-key configuration value**
+The configuration value for the KMS wrapper-key now accepts a simple string.
+Update your configuration as follows:
+
+.. code:: bash
+
+    crypto.private-key-store.encryption.wrapper-key-id = { str = "..."} # version 2.7
+    crypto.private-key-store.encryption.wrapper-key-id = "..." # version 2.8
+
+
+**Indexer Schema Migration and Cache Weight Configuration**
+Remove the following configuration lines related to the indexer and Ledger API server schema migration and cache weight:
+
+.. code:: bash
+
+    participants.participant.parameters.ledger-api-server-parameters.indexer.schema-migration-attempt-backoff
+    participants.participant.parameters.ledger-api-server-parameters.indexer.schema-migration-attempts
+    participants.participant.ledger-api.max-event-cache-weight
+    participants.participant.ledger-api.max-contract-cache-weight
+
+Breaking console commands
+"""""""""""""""""""""""""
+
+**Key Management Commands**
+The ``owner_to_key_mappings.rotate_key`` command is deprecated.
```

Comment:
I think this sentence creates confusion because it seems to indicate that the method was removed or significantly changed while it only require an additional parameter. Also we try to avoid references to internal types.
What about:
```
Command ``owner_to_key_mappings.rotate_key`` command was changed to avoid unwanted key rotations.
It now expects a node reference to avoid unwanted key rotations.
```